src/OpenIDConnectClient.php

```diff
@@ -786,7 +789,7 @@ private function verifyRSAJWTsignature($hashtype, $key, $payload, $signature) {
             "  <Modulus>" . b64url2b64($key->n) . "</Modulus>\r\n" .
             "  <Exponent>" . b64url2b64($key->e) . "</Exponent>\r\n" .
             "</RSAKeyValue>";
-	if(class_exists('Crypt_RSA')) {
+	if(class_exists('Crypt_RSA', false)) {
```

Comment:
the second argument is "autoload". configured it to false as otherwise we combine this library with composer